assert-device

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+
+import sys
+
+device = sys.argv[1]
+
+"""Assert that the device identifier is the given string."""
+cmd = ('assert(' +
+       ' || '.join(['getprop("ro.product.device") == "%s" || getprop("ro.build.product") == "%s"'
+                     % (i, i) for i in device.split(",")]) +
+       ' || abort("This package is for device: %s; ' +
+       'this device is " + getprop("ro.product.device") + ".");' +
+       ');') % device
```

Comment:
the resulting python output ends with ``assert(...,abort(...););`` but if recovery can chew on a semicolon-terminated function call as another function's argument, then I guess it's all ok (since CM scripts do that themselves :D )

I'll merge this, in complete faith of CM's wisdom :) Thanks!